scripts/reloader.bash

```diff
@@ -19,23 +19,30 @@ if [[ "$1" != "skip" ]] && [[ -d "$BASH_IT/enabled" ]]; then
       _log_debug "Loading all enabled components..." ;;
   esac
 
-  for _bash_it_config_file in $(sort <(compgen -G "$BASH_IT/enabled/*${_bash_it_config_type}.bash")); do
```

Comment:
I'm extremely curious why this construct is here in the first place. Why use compgen to glob the file path, then sort it? I assume this is something to do with ensuring that files are loaded in alphabetical order (priority number before the ---), but...isn't that true with normal globbing?